netutils/ftpd/ftpd.c

```diff
@@ -1809,9 +1809,6 @@ static int ftpd_stream(FAR struct ftpd_session_s *session, int cmdtype)
 #if defined(O_LARGEFILE)
   oflags |= O_LARGEFILE;
 #endif
-#if defined(O_BINARY)
```

Comment:
Let's check the original canutils code. If it contains this code, then it is better to keep it even if the binary flag is zero

Reply:
Sure

Reply:
Posted comment on a wrong line, but still... I've checked can-utils and do not see any reference to O_BINARY